src/Adapter/MSTest.TestAdapter/TestingPlatformAdapter/TestApplicationBuilderExtensions.cs

```diff
@@ -34,7 +45,7 @@ public static void AddMSTest(this ITestApplicationBuilder testApplicationBuilder
         testApplicationBuilder.AddRunSettingsEnvironmentVariableProvider(extension);
         testApplicationBuilder.RegisterTestFramework(
             serviceProvider => new TestFrameworkCapabilities(
-                new VSTestBridgeExtensionBaseCapabilities(),
+                new MSTestCapabilities(),
```

Comment:
Do we really need this? Removing VSTestBridge will not be enough?

Reply:
@fhnasser We still use the bridge, but only don't want to use vstestProvider.